app/assets/javascripts/covid19/src/webapp/components/structures-table/cells/LigandsCell.tsx

```diff
@@ -37,21 +43,61 @@ export const LigandsCell: React.FC<CellProps> = React.memo(props => {
         });
     }, [row.ligands]);
 
+    React.useEffect(() => {
+        row.ligands.forEach(ligand =>
+            compositionRoot.ligands.getIDR.execute(ligand.id).run(
```

Comment:
The use case should do all the task, so it gets `ids` instead of `id`. And inside it does a Future.sequential (or parallel) + merge of the data.